cmd/minikube/cmd/start.go

```diff
@@ -77,6 +77,16 @@ var (
 	apiServerNames   []string
 	apiServerIPs     []net.IP
 	hostRe           = regexp.MustCompile(`^[^-][\w\.-]+$`)
+	slowK8sVersions  = map[string]struct{}{
```

Comment:
how about moving this to a package like `reason` 
Similar to 
https://github.com/medyagh/minikube/blob/b45c782df392d90613d6e8bcc94e029cd13128bb/pkg/minikube/reason/match.go#L54
```
// MatchKnownIssue returns a known issue from an error on an OS
func MatchKnownIssue(r Kind, err error, goos string) *Kind {
```


if there is no issue it will return Nil

if there is an issue it will return a K8sIssueKind
```
type K8sIssueKind struct {
	IsssueSummar string // For example Causes signigicant slowdown in creating cluster

	// URL is a reference URL for more information
	URL string
	// Do not attempt to match this reason to a specific known issue
	NoMatch bool
}

```

Reply:
This feels like overengineering to me, unless we intend on abstract all possible issues that can stem from Kubernetes itself.

Reply:
for sake of maintainability of minikube thats the right thing to do, since we have a package for matching issues,
and will make our start.go not longer than currently is, it would make it easier for future contributors to see what is going on

Reply:
makes sense, done